shap/explainers/tree.py

```diff
@@ -46,12 +46,22 @@ class TreeExplainer(Explainer):
     implementations either inside the externel model package or in the local compiled C extention.
     """
 
-    def __init__(self, model, **kwargs):
+    def __init__(self, model, direct = False, output_type = "margin", 
```

Comment:
We should add an option here about what kind of dependence assumptions we are making: `feature_dependence=independent`, `feature_dependence=global_tree_dependent`, or `feature_dependence=per_tree_dependent`